themes/classic/templates/catalog/_partials/product-details.tpl

```diff
@@ -50,7 +50,11 @@
           <dl class="data-sheet">
             {foreach from=$product.features item=feature}
               <dt class="name">{$feature.name}</dt>
-              <dd class="value">{$feature.value}</dd>
+              {if $feature.id_feature === '4'}
```

Comment:
Sorry guys but there's always a choice, you can't hardcode id of feature when this feature is most of the time removed by `pscleaner` module after first installation of PrestaShop